### DISCUSSION TITLE
Installation Failed

### DISCUSSION THREAD

**Discussion by alimp5** [projectdiscovery/interactsh#43] [Q&A] [answered]
```
┌──(root💀milani)-[~]
└─# go get -v github.com/projectdiscovery/interactsh/cmd/interactsh-client                                   127 ⨯

github.com/pierrec/lz4 (download)
cannot find package "github.com/pierrec/lz4/v4" in any of:
        /usr/lib/go-1.15/src/github.com/pierrec/lz4/v4 (from $GOROOT)
        /root/go/src/github.com/pierrec/lz4/v4 (from $GOPATH)

```

https://github.com/pierrec/lz4/v4

**Comment by geeknik** [comment]
Unable to reproduce this. In fact, I don't see a reference to lz4 in any of the [interactsh-client](https://github.com/projectdiscovery/interactsh/blob/main/cmd/interactsh-client/main.go) [source code](https://github.com/projectdiscovery/interactsh/blob/main/pkg/client/client.go). You could try this

`go get -v github.com/pierrec/lz4 && go get -v github.com/projectdiscovery/interactsh/cmd/interactsh-client`

**Comment by ehsandeep** [comment]
I guess this is something related to issue with go setup.

**Comment by Mzack9999** [comment, marked as answer]
It's an issue related to your go module local cache, try to clean it up with:

```
go clean --modcache
```
